project-base/storefront/components/Layout/Header/MenuIconic/MenuIconicElements.tsx

```diff
@@ -11,24 +11,26 @@ export const MenuIconicItem: FC<{ title?: string }> = ({ children, className, ti
 
 type MenuIconicItemLinkProps = { onClick?: () => void; href?: string; title?: string; type?: PageType };
 
+const menuIconicSubItemLink = 'flex items-center h-14 px-5 py-3 text-sm text-text no-underline font-semibold';
```

Comment:
Could you please change name similar to `menuIconicItemLinkTwClass`

Reply:
This classname is already used on line 32. 
But you probably want me to add TwClass on the end of classname. So I did (;

Reply:
exactly!